index.js

```diff
@@ -77,11 +77,13 @@ const createElement = tagName => {
 	return document.createElement(tagName);
 };
 
-const setAttribute = (tagName, el, name, value) => {
-	if (isSVG(tagName)) {
-		el.setAttribute(name, value);
+const setAttribute = (el, name, value) => {
+	// Naive support for xlink namespace
+	// Full list: https://github.com/facebook/react/blob/1843f87/src/renderers/dom/shared/SVGDOMPropertyConfig.js#L258-L264
+	if (/^xlink[AHRST]/.test(name)) {
+		el.setAttributeNS('http://www.w3.org/1999/xlink', name, value);
 	} else {
-		el.setAttributeNS(null, name, value);
```

Comment:
From what I saw in the browsers, attributes don't have a namespace (unless it's xlink like here). Try yourself with `someSvgElement.attributes[0].namespaceURI`

Also `setAttributeNS(null)` doesn't set any namespace anyway; setting a null namespace is exactly how `setAttribute` works